### ISSUE TITLE
No poster

### ISSUE THREAD

**Issue by ink79** [seanap/Audiobooks.bundle#2]
the plugin suddenly stop adding poster to audiobook tagging work perfectly, i have changed to the original plugin which still add poster to books

**Comment by seanap** [comment]
I'm running the most recent version and I'm not seeing an issue on my end.  The code to set the poster has not been changed or updated. Make sure mp3tag is up to date and make sure you are using the latest .src file in my repo. 

Do you have an example of a book you are looking up that it can't find the poster?

**Comment by ink79** [comment]
i have tried it with two server same issue, cant get poster with your version but when i replace the plugin with original & do metadata scan able to get poster

**Comment by ahillebrandt** [comment]
> i have tried it with two server same issue, cant get poster with your version but when i replace the plugin with original & do metadata scan able to get poster

i just switched to the macr0dev code and the posters are loading now.
switched to yours because of the great guide, but now... no images... strange

**Comment by seanap** [comment]
Alright, I figured it out.  I specifically took away the agents ability to set the Cover (and forgot that I did that, sorry!).

Since we already went through all the work of tagging the file, I wanted Plex to set the Cover, Title, Author, and Genre's from the file and prevent the Agent from changing those fields.  

There shouldn't be any difference between the cover of the book when scrapped in mp3tag vs when scrapped in the agent.  But the reason I prevented the Agent from setting the cover was because if you set your own poster in the file I didn't want that cover to be overwritten.

If you do not have your files tagged, and you want the agent to behave more like the original then open up the _init_.py in Notepad++

If you want the agent to set the genre's:
Uncomment Lines 580, 585, 586

If you want the agent to set the Title and Author:
Uncomment Lines 593 and 594

If you want the agent to set the Cover:
Uncomment lines 590 and 591
